Week1/homework/js-exercises/8_typeChecker.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+let firstData = "Messi";
+let secondData = "Ronaldo";
+let thirdData = [];
+let fourthData ={};
+console.log(typeof firstData === typeof secondData);
+console.log(typeof firstData === typeof thirdData);
+console.log(typeof firstData === typeof fourthData);
+console.log(typeof secondData === typeof thirdData);
+console.log(typeof secondData === typeof fourthData);
+console.log(typeof thirdData === typeof fourthData);
+
```

Comment:
Lines 6-11 are redundant, as you already compared the types in the lines below.
But you could rewrite them to answer this request:

> 8.4 Write 2 console.log statements to log the type of 2 variables, each with a different data type